sdk/ai/azure-ai-inference/azure/ai/inference/aio/_patch.py

```diff
@@ -823,8 +1005,56 @@ def __init__(
         self, endpoint: str, credential: Union[AzureKeyCredential, "AsyncTokenCredential"], **kwargs: Any
     ) -> None:
         self._model_info: Optional[_models.ModelInfo] = None
+        self._model_extras: Optional[Dict[str, Any]] = None
+        self._dimensions: Optional[int] = None
+        self._encoding_format: Optional[Union[str, _models.EmbeddingEncodingFormat]] = None
+        self._input_type: Optional[Union[str, _models.EmbeddingInputType]] = None
+        self._model: Optional[str] = None
         super().__init__(endpoint=endpoint, credential=credential, **kwargs)
 
+    def with_defaults(
+        self,
+        *,
+        model_extras: Optional[Dict[str, Any]] = None,
+        dimensions: Optional[int] = None,
+        encoding_format: Optional[Union[str, _models.EmbeddingEncodingFormat]] = None,
+        input_type: Optional[Union[str, _models.EmbeddingInputType]] = None,
+        model: Optional[str] = None,
+    ) -> "ImageEmbeddingsClient":
+        """Set defaults for all following image embeddings operations.
+
+        :keyword model_extras: Additional, model-specific parameters that are not in the
+         standard request payload. They will be added as-is to the root of the JSON in the request body.
+         How the service handles these extra parameters depends on the value of the
+         ``extra-parameters`` request header. Default value is None.
+        :paramtype model_extras: dict[str, Any]
+        :keyword dimensions: Optional. The number of dimensions the resulting output embeddings should
+         have.
+         Passing null causes the model to use its default value.
+         Returns a 422 error if the model doesn't support the value or parameter. Default value is
+         None.
+        :paramtype dimensions: int
+        :keyword encoding_format: Optional. The desired format for the returned embeddings.
+         Known values are:
+         "base64", "binary", "float", "int8", "ubinary", and "uint8". Default value is None.
+        :paramtype encoding_format: str or ~azure.ai.inference.models.EmbeddingEncodingFormat
+        :keyword input_type: Optional. The type of the input.
+         Returns a 422 error if the model doesn't support the value or parameter. Known values are:
+         "text", "query", and "document". Default value is None.
+        :paramtype input_type: str or ~azure.ai.inference.models.EmbeddingInputType
+        :keyword model: ID of the specific AI model to use, if more than one model is available on the
+         endpoint. Default value is None.
+        :paramtype model: str
+        :return: This ImageEmbeddingsClient object, with defaults set.
+        :rtype: ~azure.ai.inference.aio.ImageEmbeddingsClient
+        """
+        self._model_extras = model_extras
+        self._dimensions = dimensions
+        self._encoding_format = encoding_format
+        self._input_type = input_type
+        self._model = model
+        return self
```

Comment:
We should return a new client instance with the merged settings.

Reply:
Done!